catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsManageCommandTest.java

```diff
@@ -0,0 +1,166 @@
+
+import static org.codice.ddf.catalog.ui.security.Constants.SYSTEM_TEMPLATE;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.refEq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.ImmutableList;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.operation.DeleteRequest;
+import ddf.catalog.operation.DeleteResponse;
+import ddf.catalog.operation.impl.DeleteRequestImpl;
+import ddf.catalog.source.IngestException;
+import ddf.catalog.source.SourceUnavailableException;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Collections;
+import java.util.List;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SearchFormsManageCommandTest {
+  private @Mock CatalogFramework catalogFramework;
+
+  private @Mock EndpointUtil endpointUtil;
+
+  private @Mock DeleteResponse deleteResponse;
+
+  private @Mock Metacard mockMetacard1;
+
+  private @Mock Metacard mockMetacard2;
+
+  private static ByteArrayOutputStream buffer;
+
+  private static PrintStream realSystemOut;
+
+  private SearchFormsManageCommand cmd;
+
+  @BeforeClass
+  public static void setupOutput() {
+    interceptSystemOut();
+  }
+
+  @Before
+  public void setup() throws SourceUnavailableException, IngestException {
+    when(mockMetacard1.getId()).thenReturn("1");
+    when(mockMetacard2.getId()).thenReturn("2");
+    when(mockMetacard1.getTitle()).thenReturn("title1");
+    when(mockMetacard2.getTitle()).thenReturn("title2");
+    when(deleteResponse.getDeletedMetacards()).thenReturn(ImmutableList.of(mockMetacard1));
+    when(catalogFramework.delete(any(DeleteRequest.class))).thenReturn(deleteResponse);
+    when(endpointUtil.getMetacardListByTag(SYSTEM_TEMPLATE))
+        .thenReturn(ImmutableList.of(mockMetacard1, mockMetacard2));
+
+    cmd = new SearchFormsManageCommand();
+    cmd.setEndpointUtil(endpointUtil);
+    cmd.setCatalogFramework(catalogFramework);
+  }
+
+  @After
+  public void resetBuffer() {
+    buffer.reset();
+  }
+
+  @AfterClass
+  public static void closeOutput() throws IOException {
+    System.setOut(realSystemOut);
+    buffer.close();
+  }
+
+  @Test
+  public void testLoadingEmptyMetacards() {
+    when(endpointUtil.getMetacardListByTag(SYSTEM_TEMPLATE)).thenReturn(Collections.emptyList());
+    cmd = new SearchFormsManageCommand();
```

Comment:
Is this and the following two lines necessary? This setup is already done in the `@Before` method.